operations/mimir-mixin/alerts/alerts.libsonnet

```
@@ -221,6 +221,42 @@
             message: '{{ $labels.job }}/%(alert_instance_variable)s has a number of mmap-ed areas close to the limit.' % $._config,
           },
         },
+      ] + [
+        {
+          alert: $.alertName('RingMembersMismatch'),
+          expr: |||
+            (
+              avg by(%(alert_aggregation_labels)s) (sum by(%(alert_aggregation_labels)s, %(per_instance_label)s) (cortex_ring_members{name=~"%(component)s",job=~"%(job)s"}))
+              != sum by(%(alert_aggregation_labels)s) (up{job=~"%(job)s"})
+            )
+            and
+            (
+              count by(%(alert_aggregation_labels)s) (cortex_build_info) > 0
+            )
+          ||| % {
+            alert_aggregation_labels: $._config.alert_aggregation_labels,
+            per_instance_label: $._config.per_instance_label,
+            component: component_job[1],
+            job: component_job[2],
```

I'm confused. Shouldn't be:

```suggestion
            component: component_job[0],
            job: component_job[1],
```

---

As per the description of the array below, this is correct as `component_job[1]` is the regex for the component, and `component_job[2]` is the regex for the job.